BraveWallet/Crypto/Accounts/AccountCoinTypesView.swift

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import SwiftUI
+import BraveCore
+import Strings
+
+struct AccountCoinTypesView: View {
```

Comment:
This view isn't used anymore

Reply:
fixed in [2691cba](https://github.com/brave/brave-ios/pull/5651/commits/2691cba17d96174b245a1af0e21327720ccf2f59)